graphalyzer-server/src/app.py

```diff
@@ -36,7 +36,7 @@ def load_edges(graphlocation: str, graphid: str):
     from py2neo import Graph
     from py2neo.packages.httpstream import http
     # Increase timeout because it can take a really long time.
-    http.socket_timeout = sys.maxsize
+    http.socket_timeout = 9999
```

Comment:
I would think it'd be okay to keep it as that for now, and tweak it when we get some data on the average, worst case time taken